crates/forge/src/init.rs

```diff
@@ -61,6 +63,17 @@ fn add_target_to_toml(path: &Path) -> Result<()> {
     Ok(())
 }
 
+fn set_cairo_edition(path: &Path, cairo_edition: &str) -> Result<()> {
+    let config_file = fs::read_to_string(path)?;
+    let mut doc = config_file.parse::<Document>().expect("invalid document");
```

Comment:
I think this parsing should be moved to `run` or a separate function so we don't parse Scarb.toml twice (here and in `add_target_to_toml`)